telebot/async_telebot.py

```diff
@@ -6237,7 +6232,18 @@ async def reply_to(self, message: types.Message, text: str, **kwargs) -> types.M
         :return: On success, the sent Message is returned.
         :rtype: :class:`telebot.types.Message`
         """
-        return await self.send_message(message.chat.id, text, reply_to_message_id=message.message_id, **kwargs)
+        if kwargs:
+            reply_parameters = kwargs.pop("reply_parameters", None)
+        else:
+            reply_parameters = None
+        if not reply_parameters:
+            reply_parameters = types.ReplyParameters(message.message_id)
+
+        if "allow_sending_without_reply" in kwargs:
```

Comment:
same here. let's check for conflicting parameters

Reply:
+